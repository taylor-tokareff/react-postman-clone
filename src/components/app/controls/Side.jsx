import React from 'react';
import PropTypes from 'prop-types';


const Side = ({ u, m }) => (
  <section>
    <h3>{m}</h3>
    <p>{u}</p>
  </section>
);

Side.propTypes = {
  u: PropTypes.string.isRequired,
  m: PropTypes.string.isRequired
};

export default Side;
