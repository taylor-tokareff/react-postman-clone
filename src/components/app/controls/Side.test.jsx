import React from 'react';
import { render } from '@testing-library/react';
import Side from './Side';

describe('Side component', () => {
  it('renders url and method', () => {
    const { asFragment } = render(
      <Side
        u="placecage.com"
        m="GET"
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
