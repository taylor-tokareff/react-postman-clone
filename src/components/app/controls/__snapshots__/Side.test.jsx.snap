// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side component renders url and method 1`] = `
<DocumentFragment>
  <section>
    <h3>
      GET
    </h3>
    <p>
      placecage.com
    </p>
  </section>
</DocumentFragment>
`;
